src/Blosc.jl

```diff
@@ -200,8 +210,8 @@ function compressor_info(name::AbstractString)
                 (Cstring,Ptr{Ptr{Cchar}},Ptr{Ptr{Cchar}}),
                 name, lib, ver)
     ret < 0 && error("Error retrieving compressor info for $name")
-    lib_str = bytestring(lib[1]); Libc.free(lib[1])
-    ver_str = bytestring(ver[1]); Libc.free(ver[1])
+    lib_str = _bytestring(lib[1]); Libc.free(lib[1])
```

Comment:
These can be updated to `lib_str = unsafe_wrap(Compat.String, lib[1], true)`, which eliminates the need to call `free`.